corehq/apps/sms/models.py

```diff
@@ -1632,7 +1632,7 @@ def get_sms_install_link(cls, domain, app_id):
         it gives the user the option to install the given app.
         """
         app_info_url = cls.get_app_info_url(domain, app_id)
-        return '[commcare app - do not delete] %s' % base64.b64encode(app_info_url)
+        return '[commcare app - do not delete] %s' % base64.b64encode(app_info_url.encode('utf-8'))
```

Comment:
`.decode('utf-8')` is needed on the end yet, right?

Reply:
👍 updated